### DISCUSSION TITLE
How Apple signin really works on local

### DISCUSSION THREAD

**Discussion by Emiliano-Bucci** [nextauthjs/next-auth#4233] [Help]
Hi! I'm trying to test the Apple signin following this guide -> https://next-auth.js.org/providers/apple
I'm not using the popup (so the user is redirected to https://appleid.apple.com/auth/authorize?...), and i've added the dev.example.com url also in the apple page. But at the end of the process, i'm being redirected to dev.example.com, so basically i can't test it locally. Am i missing something? Thanks

**Comment by nmandal** [comment]
@Emiliano-Bucci did you ever get this working? Have been stuck for a number of weeks - happy to pair on a solution.
